docs/the-new-architecture/landing-page.md

```diff
@@ -3,11 +3,26 @@ id: landing-page
 title: Introduction
 ---
 
-This section is the entry point for the new guide’s documentation.
-It contains some basic information about the New Architecture: its pillars, the version from which it has been made available and other very high-level information. Then, it should present the guide’s structure itself.
+Starting in version 0.68, React Native provides a New Architecture that improves several aspects of how JavaScript code and host platforms can integrate, such as:
 
-This section should also contains a sort of Driving guide based on the user use-cases:
+- synchronously measure and render React Native views at the same time as native views
+- support for prioritization of events that update the app's layout, like user interactions
+- improved type safety and consistency of communication between JavaScript and native code
```

Comment:
You can pick some other benefits from [here](https://github.com/facebook/react-native-website/pull/3043). It also contains a slightly more detailed explanation about why we are migrating to the new architecture.

Reply:
The explanation in that document is thorough, but concise. Given that it's not very long, I'm wondering how much is worth repeating vs. just expressing something along the lines of "The New Architecture provides developers more tools for creating performant and responsive apps. Check out 'Why a New Architecture' for more information."

Reply:
> "The New Architecture provides developers more tools for creating performant and responsive apps. Check out 'Why a New Architecture' for more information."

Great suggestion. Go for it! :+1: